driver-examples/tycho/.gitignore

```diff
@@ -0,0 +1,2 @@
+!.classpath
```

Comment:
I think it's better to provide these files. mvn eclipse:eclipse is not capable of handling plugin projects, importing existing Maven projects neither.